cmd/traffic/cmd/manager/internal/watchable/generic.tmpl.go

```diff
@@ -4,31 +4,29 @@
 package watchable
 
 import (
-	"context"
-	"sync"
+    "context"
+    "sync"
 
-	"google.golang.org/protobuf/proto"
-
-	"VALPKG"
```

Comment:
This change is incorrect. Please remove it.

Reply:
Done